### ISSUE TITLE
Migrate away from standalone X server

### ISSUE THREAD

**Issue by DemiMarie** [QubesOS/qubes-issues#7914]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release (if applicable)

R4.1

### Brief summary

The standalone X server is deprecated and (for all intents and purposes) unmaintained.  Xwayland’s rootful mode can be used if a full X11-based desktop environment is needed (XFCE), but KDE can and should use the native Wayland session.

**Comment by jamke** [comment]
> Xwayland’s rootful mode can be used if a full X11-based desktop environment is needed (XFCE), but KDE can and should use the native Wayland session.

Does it indirectly implies a possibility of kde being a default replacement of xfce again? :)

**Comment by DemiMarie** [comment]
> > Xwayland’s rootful mode can be used if a full X11-based desktop environment is needed (XFCE), but KDE can and should use the native Wayland session.
> 
> Does it indirectly implies a possibility of kde being a default replacement of xfce again? :)

That one is up to @marmarek.

**Comment by ben-grande** [comment]
@DemiMarie Isn't this a requirement for the Wayland port, meaning, unforgettable task? Is this issue necessary? @valpackett

**Comment by DemiMarie** [comment]
@ben-grande Yes, it’s a requirement of the Wayland port.  I’m fine with closing this issue.
